src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -1060,6 +1060,12 @@ object SymDenotations {
       else defn.RootClass
     }
 
+    /** Is this denotation public ? */
+    def isPublic(implicit ctx: Context): Boolean =
+      // private-qualified denotations do not have the Private flag set,
+      // see `ParserCommon#normalize`
+      !this.is(Protected | Private) && !privateWithin.exists
```

Comment:
A method that is `private[package]` is public in the bytecode but it's not public in the Scala language. `public` means accessible from everywhere, `private[package]` is only accessible in some cases (if you're in the same package).
Anyone who writes their own phase for dotty and wants to check if a symbol is public in the Scala language will want `isPublic` to return false for something which is `private[package]`, the fact that it is public in the bytecode is an implementation detail.

Anyway, I don't want to fight on this, I'll just put `def isPublic` in sbt and leave the rest alone.